src/test/resources/function-tests.json

```diff
@@ -1046,6 +1046,66 @@
       "query": "max(null, 0)",
       "input" : "null",
       "output": "null"
+   },
+   {
+      "query" : "hash-int(.)",
+      "input" : "\"\"",
+      "output" : "1088"
+   },
+   {
+      "query" : "hash-int(.)",
+      "input" : "\"test\"",
+      "output" : "1083642606"
+   },
+   {
+      "query" : "hash-int(.)",
+      "input" : "{}",
+      "output" : "119320"
+   },
+   {
+      "query" : "hash-int(.)",
+      "input" : "{    }",
+      "output" : "119320"
+   },
+   {
+      "query" : "hash-int(null)",
```

Comment:
Confusing to have the input in the expression, but what looks like the input on the next line